packages/expo-module-template-local/expo-module.config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "platforms": ["ios", "android", "web"],
+  "platforms": ["ios", "tvos", "android", "web"],
```

Comment:
Just to let you know: `ios`, `tvos`, `macos` are all deprecated in favor of just `apple`. Sadly we can't remove `ios` from the template yet because it would break compatibility with SDK 49 (we shouldn't force module authors to require SDK 50).